src/main/java/cn/nukkit/entity/Entity.java

```diff
@@ -1804,6 +1804,7 @@ public boolean onInteract(Player player, Item item, Vector3 clickedPos) {
         return onInteract(player, item);
     }
 
+    @Deprecated
```

Comment:
Any reason why this was removed? Seems like it's out of the scope of this PR.

Reply:
This was replaced with `onInteract(Player player, Item item, Vector3 clickedPos)` and plugins should override that instead